api/src/main/java/org/pmiops/workbench/cohortbuilder/querybuilder/DemoQueryBuilder.java

```diff
@@ -85,6 +86,10 @@ public QueryJobConfiguration buildQueryJobConfig(QueryParameters parameters) {
                     throw new IllegalArgumentException("Dec must provide a value of: " + DECEASED);
                 }
 
+            } else if (key.equals(DEMOTYPE.ETH.name())) {
+                final String namedParameter = key.toLowerCase() + getUniqueNamedParameterPostfix();
+                queryParts.add(DEMO_ETH.replace("${eth}", "@" + namedParameter));
+                queryParams.put(namedParameter, QueryParameterValue.array(paramMap.get(key).stream().toArray(Long[]::new), Long.class));
```

Comment:
Not hugely important, but you could be a little safer by streaming through a filter to ensure everything really is a Long value, then map to cast it, and finally to an array.